libs/wingc/src/type_check.rs

```diff
@@ -1875,7 +1876,7 @@ impl<'a> TypeChecker<'a> {
 		}
 	}
 
-	pub fn type_check_scope(&mut self, scope: &Scope) {
+	pub fn type_check_scope(&mut self, scope: &'a Scope) {
```

Comment:
I think the conventional Rust style is to omit lifetimes unless they can't be inferred by the compiler